tracer/tracer.go

```diff
@@ -2,16 +2,47 @@ package tracer
 
 import (
 	"log"
+	"os"
+	"strings"
 	"time"
 
 	"context"
+	"fmt"
 )
 
 const (
-	defaultDeliveryURL = "http://localhost:7777/v0.1/spans"
-	flushInterval      = 2 * time.Second
+	defaultTraceTarget  = "localhost:7777"
+	defaultTraceEnabled = true
+	traceTargetVar      = "DATADOG_TRACE_TARGET"
```

Comment:
Actually they're two different things (so they can potentially run in two different hosts) but the plan is to have one single agent that will handle both.